employees/common/strings.py

```diff
@@ -110,4 +110,4 @@ class ReportValidationStrings(NotCallableMixin, Enum):
 
 class MonthNavigationText(NotCallableMixin, Enum):
     SWITCH_MONTH = _("Go")
-    RECENT_MONTH = _("Recent month")
+    RECENT_MONTH = _("Current month")
```

Comment:
Variable should not be change to `CURRENT_MONTH` as well ?

Reply:
Good point:)